.travis.yml

```diff
@@ -26,7 +26,10 @@ install:
   - pip install https://github.com/pysal/libpysal/archive/master.zip
   # Pysal dependencies removed, therefore need spreg master import
   - pip install https://github.com/pysal/spreg/archive/master.zip
-  - pip install esda giddy mapclassify palettable
+  # `libpysal` api change requires esda and giddy import from masterbranch
+  - pip install https://github.com/pysal/esda/archive/master.zip
```

Comment:
Looks good to me! Since new versions of `esda`, `giddy` and `mapclassify` have been released on pypi. Testing against the released version should also pass now.